### PR TITLE
V6/add telegram link to the footer

### PR DESCRIPTION
**Pull request by hsurf22**
### Devop

* \[x] Updated CHANGELOG.md
* \[x] Add PR label
  - Add telegram link to footer

### CODE REVIEW

**Comment by olgakup** [comment]
@hsurf22 please fix build

**Comment by hsurf22** [comment]
> @hsurf22 please fix build

Fixed!

File: src/views/components-default/TheDefaultFooter.vue [hunk 1]
```diff
@@ -348,7 +348,11 @@ export default {
           },
           { label: 'MEW topia', link: 'https://www.mewtopia.com/' },
           { label: 'Press Kit', routerLink: 'PressKit' },
-          { label: 'Security Policy', routerLink: 'SecurityPolicy' }
+          { label: 'Security Policy', routerLink: 'SecurityPolicy' },
```
Comment (olgakup) [inline src/views/components-default/TheDefaultFooter.vue:351]: Sorry, could not see where the link went. Its actually supposed to be with the social media links on the bottom:
Screen Shot 2021-06-29 at 10 42 49 AM

**Review by olgakup — CHANGES_REQUESTED** [review]

File: src/views/components-default/TheDefaultFooter.vue (hunk 1 above)
Comment (hsurf22) [reply src/views/components-default/TheDefaultFooter.vue:340]: fixed

Screenshot from 2021-07-02 19-19-43

**Review by gamalielhere — APPROVED** [review]

**Review by gesseekur — APPROVED** [review]

**Pull request merged**